extension/src/test/DocumentFunctions.test.ts

```diff
@@ -0,0 +1,22 @@
+import * as assert from "assert";
+import * as vscode from "vscode";
+import * as DocumentFunctions from "../DocumentFunctions";
+
+suite("DocumentFunctions", function () {
+  test("openTextFileWithSelectionOnLineNo", function () {
+    assert.ok(DocumentFunctions.openTextFileWithSelectionOnLineNo("", 0));
```

Comment:
Why do we not use `assert.ok(condition)` instead of `assert.strictEqual(condition, true,'message')` (or `assert.equal(condition, true,'message')`) all over the place? 😁

Reply:
I realize that we always should use the `message` parameter of the assert functions. 
`assert.ok(condition, message)`
Otherwise it can be challenging to find out exactly what has failed.

Reply:
Yes, think we should switch out truthy assertions for `assert.ok` it would be an improvement :) https://doc.codingdict.com/nodejs-ref/assert.html#assert_assert_ok_value_message

Reply:
How many times have I debugged the wrong assertion due to misreading or duplicating messages when copy/pasting... waaay to many :D